### PR TITLE
Request catalog search app name

### PR DESCRIPTION
**Pull request by loganintech**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,9 @@
 
 .PHONY: gen
 gen:
-	speakeasy generate sdk -s openapi.yaml -o . -d
\ No newline at end of file
+	@echo "Generating SDK"
+	curl -sSL https://insulator.conductor.one/api/v1/openapi.yaml > ./openapi.yaml
```
Comment (ggreer) [inline Makefile:5]:
```suggestion
	curl -sSL -o openapi.yaml https://insulator.conductor.one/api/v1/openapi.yaml
```

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+	speakeasy generate sdk -s openapi.yaml -o . -d
+	rm openapi.yaml
+	@echo "Fixing Permissions"
+	find * -type f -perm '+rwx' | xargs chmod -x
```
Comment (ggreer) [inline Makefile:9]: It'd be nice to leave a comment here explaining why we do this, so we can remove it when speakeasy fixes their CLI.

**Review by ggreer — APPROVED** [review]
A couple minor nitpicks. Feel free to fix them or just merge without changing stuff.

**Review by ggreer — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -2,8 +2,10 @@
 .PHONY: gen
 gen:
 	@echo "Generating SDK"
-	curl -sSL -o openapi.yaml https://insulator.conductor.one/api/v1/openapi.yaml
+	curl -sSL https://insulator.conductor.one/api/v1/openapi.yaml -o openapi.yaml2
```
Comment (ggreer) [inline Makefile:5]: Wait. .yaml2?

Comment (loganintech) [reply Makefile:5]: Whoops

Comment (loganintech) [reply Makefile:5]: #31

**Pull request merged**